src/main/java/com/example/ordermanagement/controller/KafkaOrderCreatedController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.ordermanagement.controller;
+
+import com.example.ordermanagement.DTO.OrdersDto;
+import com.example.ordermanagement.service.kakfa.ProducerService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Slf4j
+@RequestMapping(value = "orders")
+public class KafkaOrderCreatedController {
```

Comment:
FYI, You can do this in orderController only

Reply:
Yes, I thought I should keep it separate so that evaluation is easier, I will put it into one controller only